indico/htdocs/js/indico/modules/surveys.js

```diff
@@ -53,6 +54,21 @@
         $('.js-add-question-dropdown').dropdown();
     }
 
+    function setupSubmissionButtons() {
+        $('.js-delete-submission').on('indico:confirmed', function(e) {
+            var $this = $(this);
+            $.ajax({
+                url: $this.data('href'),
+                method: $this.data('method'),
+                complete: IndicoUI.Dialogs.Util.progress(),
+                error: handleAjaxError,
+                success: function(data) {
+                    location.reload(true);
```

Comment:
And what in case when there are no rows left? Leave the list empty or insert a row with a proper message?